dev-requirements.txt

```diff
@@ -0,0 +1,9 @@
+coverage==4.5.2
+coveralls==1.5.1
+pytest==4.1.0
+pytest-cov==2.6.1
+Sphinx==1.8.5
+sphinx-bootstrap-theme==0.6.5
+sphinxcontrib-fulltoc==1.2.0
+sphinxcontrib-websupport==1.1.0
+twine==1.13.0
```

Comment:
Mind adding this dependency also:

> attrs==19.1.0

I just ran the tests with a fresh virtual environment and it seems a dependency of pytest was updated early October and breaks things unless you explicitly add the previous version... :upside_down_face:

Reply:
Added! Thanks.